drivers/filter_wheel/ioptron_wheel.cpp

```diff
@@ -0,0 +1,331 @@
+/*******************************************************************************
+  Copyright(c) 2024 Joe Zhou. All rights reserved.
+
+  iOptron Filter Wheel
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Library General Public
+ License version 2 as published by the Free Software Foundation.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Library General Public License for more details.
+
+ You should have received a copy of the GNU Library General Public License
+ along with this library; see the file COPYING.LIB.  If not, write to
+ the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ Boston, MA 02110-1301, USA.
+*******************************************************************************/
+
+#include "ioptron_wheel.h"
+
+#include "indicom.h"
+#include <connectionplugins/connectionserial.h>
+#include <cstring>
+#include <memory>
+#include <termios.h>
+
+#define iEFW_TIMEOUT 4
+
+/*
+command :
+:WMnn#          Response:  1
+get position ，nn filter num。
+
+:WP#            Response:  nn#  or  -1#
+get curr position,nn =curr position，if is moving，return -1。
+
+:WOnnsnnnnn#	Response:  1
+set offset。nn =filter num , snnnnn=offset
+
+:WFnn#          Response:     snnnnn#
+get offset 。nn = filter num , snnnnn=offset
+
+“:DeviceInfo#”		Response: “nnnnnnnnnnnn#”
+This command includes 12 digits.
+The 7th and 8th digit means model of filter wheel.
+Model number  99 is iEFW-15, 98 is iEFW18
+
+:FW1#			Response:  nnnnnnnnnnnn#
+get firmware version
+
+
+*/
+
+
+static std::unique_ptr<iEFW> iefw_wheel(new iEFW());
+
+iEFW::iEFW()
+{
+    setVersion(1, 0);
+    setFilterConnection(CONNECTION_SERIAL);
+}
+
+const char *iEFW::getDefaultName()
+{
+    return "iOptron Wheel";
+}
+
+bool iEFW::initProperties()
+{
+    INDI::FilterWheel::initProperties();
+    serialConnection->setDefaultPort("/dev/ttyUSB3");
```

Comment:
I don't think it should be set to this port. Please remove this line as this is a generic port.